src/App.vue

```diff
@@ -41,8 +38,8 @@
       <!-- Components according to route -->
       <router-view />
 
-      <!-- SBC Common Components header -->
-      <sbc-footer :aboutText=aboutText />
+      <!-- SBC Common Components footer -->
+      <SbcFooter :aboutText=aboutText />
```

Comment:
So.... I think I do prefer PascalCase for our custom components, as they stand out in VS Code and I can F12 them to directly to go to their declarations.